src/pkg/selectors/sharepoint_test.go

```diff
@@ -186,7 +186,8 @@ func (suite *SharePointSelectorSuite) TestSharePointSelector_Exclude_WebURLs() {
 
 	sel.Exclude(sel.WebURL([]string{s1, s2}))
 	scopes := sel.Excludes
-	require.Len(t, scopes, 1)
+	// @keepers: Question Should this be 2 as in check for SharePoint Lists and SharePoint Drives?
```

Comment:
Yep!  Because WebURLs are alternative to SiteIDs, they apply to both categories.

Reply:
Comment removed within [0d69ba2](https://github.com/alcionai/corso/pull/1786/commits/0d69ba2066604fb4c9e425e4f2978d3229d4cf8d)